src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md

```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
 ---
 ## Give a Background Color to a div Element
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+You can change the `background` `color` to a `div` Element (or section) by one of two ways.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<b>The First Method:</b>
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+  Create a `class` in the style brackets.
+```html
+<style>
+ .blue-background {
+  background-color: blue;
+  }
+</style>
+```
+You can then add the `class` to your `div` Element:
+```html
+  <div class="blue-background">
```

Comment:
Sorry to keep going, but could you just fix the indentation of this code block? Then we will be good to merge 😄